react/rules/RUS.js

```diff
@@ -0,0 +1,73 @@
+import { isPastDate } from '../utils/dateRules'
```

Comment:
Missing changelog entry. Vide:

https://github.com/vtex/profile-form/blob/master/CHANGELOG.md#added

Reply:
Added also in this Branch!, Thanks Felipe for your Support!